lib/sequelize.js

```diff
@@ -232,6 +233,12 @@ class Sequelize {
           }
         }
       }
+
+      // For postgres, we can use this helper to load certs directly from the
+      // connection string.
+      if (options.dialect === 'postgres' || options.dialect === 'postgresql') {
+        Object.assign(options.dialectOptions, pgConnectionString.parse(arguments[0]));
```

Comment:
Just a thought really: isn't args[0] === database?

Reply:
I don't think so - as per https://github.com/sequelize/sequelize/blob/47c4494968422585bf265063925d1662ffcd4173/lib/sequelize.js#L185

it is treated as a connection string in this conditional branch.

Reply:
Yeah I found that line confusing as well :) anyway. I don't mind either way

Reply:
Can you check if this still occurs on `main`/the latest v7 alpha? If so, can you create an issue for it?

Reply:
https://github.com/sequelize/sequelize/issues/15823

Took a while. Issue creation has a lot of requirements :)